datadog/api/base.py

```diff
@@ -124,6 +124,14 @@ def request(cls, method, path, body=None, attach_host_name=False, response_forma
                 if e.response.status_code == 404 or e.response.status_code == 400:
                     pass
                 else:
+                    if e.response.text:
+                        try:
+                            error_msg = json.loads(e.response.text).get('errors')
+                            if error_msg:
+                                raise requests.exceptions.HTTPError("{0} ({1})".format(
+                                                                    e.message, error_msg[0]))
+                        except json.JSONDecodeError:
+                            pass
```

Comment:
Sure! Some 403 responses include custom error messages. For example, if a user tries to `$ dog monitor unmute_all` when the `mute_all` setting isn't enabled, the 403 response includes the error message: `'Monitoring is not currently globally muted'`. Simply raising `HttpError`, however, just gives the user a generic `'403 Client forbidden error'`.